Automattic/jetpack
# Subscriptions: Apply the subscriber logout function globally

Related discussion: p1710420752713079/1710159889.840179-slack-C01B6KEJ5GE

Related to subscribe member login https://github.com/Automattic/jetpack/issues/34884

## Proposed changes:

It adds the clearing premium content cookie logic globally in the Memberships module instead of having it in the Subscriber Login block.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Be a logged in subscriber
* Open the content for paid members only
* Click Switch Accounts
    <img width="409" alt="image" src="https://github.com/Automattic/jetpack/assets/87168/26bcdcc8-2087-4a5f-b619-1554cda62b5d">
* Get back to the page without logging in and make sure the premium content cookie has been removed
